pom.xml

```diff
@@ -52,7 +52,7 @@
 
         <powsybl-ws-dependencies.version>2.11.0</powsybl-ws-dependencies.version>
         <!-- FIXME: powsybl-network-store modules'version is overloaded in the dependencies section.The overloads and this property below have to be removed at next powsybl-ws-dependencies.version upgrade -->
-        <powsybl-network-store.version>1.13.0</powsybl-network-store.version>
+        <powsybl-network-store.version>1.14.0-SNAPSHOT</powsybl-network-store.version>
```

Comment:
Wait for next network-store client release before merging this PR (with this new release in pom.xml)